src/components/PokemonCard.tsx

```diff
@@ -65,7 +65,8 @@ const PokemonCard: React.FC<{
       </CardHeader>
       <CardContent>
         <div className="flex flex-col space-y-6">
-          <div className="flex flex-row items-center md:items-start space-y-4 justify-around md:space-y-0">
+          <div className="flex flex-row items-center md:items-start space-y-4 justify-between md:space-y-0">
+            <div></div>
```

Comment:
Can we remove this?

Reply:
I wanted to change the parent div to space-between because IV chart was getting cut off so I wanted it to give as much width as possible by letting it extend to the far right, but I wanted to keep the space left of the image

Maybe left margin on the image would have been better?

Reply:
Changed to using margin-left